opflexagent/gbp_ovs_agent.py

```diff
@@ -577,6 +580,35 @@ def main():
     common_config.setup_logging()
     q_utils.log_opt_values(LOG)
 
+    agent_mode = cfg.CONF.OPFLEX.agent_mode
+    if agent_mode == 'dvs':
+        main_dvs()
+    else:
+        main_opflex()
+
+
+def main_dvs():
+    dvs_agent = None
+    try:
+        dvs_agent = importlib.import_module(DVS_AGENT_MODULE)
+        agent_config = dvs_agent.create_agent_config_map(cfg.CONF)
```

Comment:
It will still use the standard ml2_conf.ini configuration file, but their contents will be different. For example, the configuration that used to be in the DVS agent's specific .ini file will now be put in the ml2_conf.ini file.